### PR TITLE
docs: Add link to chromeWebSecurity read more

### PR DESCRIPTION
**Pull request by esetnik**
### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [x] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by jennifer-shehane — APPROVED** [review]

File: cli/types/cypress.d.ts [hunk 1]
```diff
@@ -3057,7 +3057,7 @@ declare namespace Cypress {
      */
     video: boolean
     /**
-     * Whether Chrome Web Security for same-origin policy and insecure mixed content is enabled. Read more about this here
+     * Whether Chrome Web Security for same-origin policy and insecure mixed content is enabled. Read more about this [here](https://docs.cypress.io/guides/guides/web-security#Disabling-Web-Security)
```
Comment (jennifer-shehane) [inline cli/types/cypress.d.ts]:
```suggestion
     * Whether Chrome Web Security for same-origin policy and insecure mixed content is enabled. Read more about this [here](https://on.cypress.io/web-security#Disabling-Web-Security)
```

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
We'd want an on link so that we can redirect this in the future.

**Comment by esetnik** [comment]
@jennifer-shehane is there anything else I need to do in order to get this merged?

**Review by jennifer-shehane — APPROVED** [review]

**Review by AtofStryker — APPROVED** [review]
to `.d.ts` render like markdown files?

**Comment by jennifer-shehane** [comment]
@AtofStryker There was a link in some other parts, so I assumed so. https://github.com/cypress-io/cypress/pull/29146/files#diff-9fde1fb2f7eaba5ea49b585c811461495bfbba88e614e900a96863ecfe6b980aL3001

**Pull request merged**